README.md

```diff
@@ -1 +1,175 @@
-# tomcat-extension-openidsso
\ No newline at end of file
+# tomcat-extension-openidsso #
```

Comment:
This is the base heading so we need to have proper naming for the heading.

Can we put "Tomcat Extension - OpenIDConnect based Single Sign On" ?